src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -73,21 +52,10 @@ public function testIfSingleTextWithDatePickerRenderedCorrectly()
         $date = new \DateTime();
 
         $element = $crawler->filter('input#form_date_example5');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e